target/linux/armvirt/64/config-5.15

```diff
@@ -99,6 +106,7 @@ CONFIG_DRM_TTM=y
 CONFIG_DRM_TTM_HELPER=y
 CONFIG_DRM_VIRTIO_GPU=y
 CONFIG_DRM_VRAM_HELPER=y
+CONFIG_ENA_ETHERNET=y
```

Comment:
x86 has a kmod package for CONFIG_ENA_ETHERNET. Do you have to compile it into the kernel?
I would prefer if you move the kmod package to package/kernel/linux/modules/netdevices.mk and share it between the targets.

Reply:
Fixed, added a change to move kmod-amazon-ena from x86 to the top level netdevices:
https://github.com/openwrt/openwrt/pull/4996/commits/1bc1899dc086534db59c9731aa4416247ec86594